src/atwho.js

```diff
@@ -131,7 +131,16 @@ define([
                                                        options.property);
                             }
                             return value;
-                        }
+                        },
+                        afterMatchFailed: function(at, $el) {
+                            if (options.useRichText) {
+                                var content = $el.html().trim();
+                                if (richText.CASE_REF_REGEX.test(content) || form.isValidHashtag(content)) {
```

Comment:
fixed `#caze #case/thing`, although in a terrible way, so this is in theory ready to merge. working on tests.